_posts/2018-09-06-how-we-use-pivotal-tracker-at-ombu-labs.markdown

```diff
@@ -2,155 +2,121 @@
-ours. This is how we like to use it to ship value with every sprint and keep
-track of our [velocity](http://wiki.c2.com/?ProjectVelocity).
+There are many things that are easier with Pivotal Tracker, as long as you are using it the right way. Some of its features are very useful for agile teams like ours. This is how we like to use it to ship value with every sprint and keep track of our [team velocity](http://wiki.c2.com/?ProjectVelocity).
 
 <!--more-->
 
-## Step 1: Filing user stories
+## Step 1: Writing user stories
 
-Every single thing has to be tracked according to the type of story. Most of the user
-stories are either *features* or *bugs*, although sometimes there will be *chores* to be
-done. If a *chore* reoccurs many times, then it's probably time to write a *feature*
-to reduce the time spent completing that *chore*.
+All tasks related to a project should be tracked through user stories. A good to way to think about user stories is that it is a high-level requirement card. A user story can be used to represent a technical requirement, a use case, a usage requirement or even chores the development team needs to complete. [Pivotal Tracker](https://www.pivotaltracker.com) gives you the ability to create 4 different types of stories:
 
-## Step 2: Describing user stories
+1. Feature
+2. Bug
+3. Chore
+4. Release
 
-Every [user story](https://www.mountaingoatsoftware.com/agile/user-stories) should
-be a simple description of a feature:
+Most stories will be either *features* or *bugs*, although sometimes there will be *chores* to be
+done. If a *chore* reoccurs many times, then it's probably time to write a *feature* to reduce the time spent completing that *chore*.
 
-```
-As a < type of user >, I want < some goal > so that < some business goal >.
-```
+Both project stakeholders and the Scrum team can request stories. If a developer sees a need for a story, they should create it. If the product owner needs to define a feature that should be added, they should create a story. However, new stories will always be added to the Icebox. Once the sprint is planned, stories being worked on will be dragged into the current iteration section of the project.
 
-The description of the user story should be very clear about the expected outcome.
-This will explain what is the exit criteria for the story. It is very helpful
-for designers and developers to understand what needs to happen for a user to
-be happy with the feature.
+In order to indicate priority, we organize the Icebox from most important to least important stories, from top to bottom. This way, everyone involved in the project has a clear picture of the priorities.
 
-This is an example from one of our recent projects ([https://audit.fastruby.io/](https://audit.fastruby.io/)):
+## Step 2: Describing user stories
+
+Every [user story](https://www.agilealliance.org/glossary/user-stories/) should be a description of a requirement and consist of a summary and a description.
```

Comment:
```suggestion
Every [user story](https://www.agilealliance.org/glossary/user-stories/) should match a requirement and consist of a summary and a description.
```

It feels weird to have description twice, I'm not 100% happy with `should match a requirement` either though haha, maybe there's a better word to put there.

Reply:
Good point. I used "representation" instead of "description"